src/ralph/sim_cards/models.py

```diff
@@ -134,3 +140,68 @@ class SIMCard(AdminAbsoluteUrlMixin, TimeStampMixin, models.Model):
 
     def __str__(self):
         return _('SIM Card: {}').format(self.phone_number)
+
+    @classmethod
+    @transition_action(
+        form_fields={
+            'user': {
+                'field': forms.CharField(label=_('User')),
+                'autocomplete_field': 'user',
+            }
+        }
+    )
+    def assign_user(cls, instances, **kwargs):
+        print(instances)
```

Comment:
just print, nothing else in this action?

Reply:
My mistake. I changed it.